lib/xboxrt/vcruntime/purecall.c

```diff
@@ -0,0 +1,37 @@
+#include <assert.h>
+#include <stdbool.h>
+#include <stdlib_ext_.h>
+#include <windows.h>
+#include <hal/debug.h>
+
+_purecall_handler current_purecall_handler = NULL;
+
+int __cdecl _purecall ()
+{
+    if (current_purecall_handler) {
+        current_purecall_handler();
+    }
+
+    DbgPrint("Pure virtual function called!\n");
+    debugPrint("Pure virtual function called!\n");
```

Comment:
I'd rather avoid these double-prints for different interfaces.

I'm worried that we'll search/replace all `debugPrint` in the future, and then we end-up with double-prints.
It also adds risk of only modifying one instance, so you'd also want to use `const char* message = "Pure virtual function called!\n";`.

If need be, we should just replicate `debugPrint` output to `DbgPrint` (might not work from interrupt handlers?) by changing debug.c.

Reply:
I removed the `DbgPrint`. It's mostly a leftover from trying my libc++ branch in xqemu with a debug bios.